features/admin/editing_presentation_slot.feature

```diff
@@ -0,0 +1,34 @@
+@managing_product_bundles
+Feature: Editing the presentation slot for a product bundle
+  In order to manage product bundle slots
+  As an administrator
+  I want to be able to set the presentation slot for a product bundle
+
+  Background:
+    Given the store operates on a single channel in "United States"
+    And the store has a product "smurf hat"
+    And the store has a product "smurf shorts"
+    And the store has a product bundle "smurf outfit"
+    And this product bundle has a slot named "headgear" with the "smurf hat" product
+    And this product bundle has a slot named "shorts" with the "smurf shorts" product
+    And the store has also a product bundle "papa smurf outfit"
+    And this product bundle has a slot named "shorts" with the "smurf shorts" product
+    And this product bundles default slot is "shorts"
```

Comment:
I think at this step 'presentation slot' is meant instead of 'default slot'?

Reply:
Sure. you're right. I'll fix it...